pkg/fission-cli/cmd/canaryconfig/command.go

```diff
@@ -76,7 +76,7 @@ func Commands() *cobra.Command {
 		RunE:    wrapper.Wrapper(List),
 	}
 	wrapper.SetFlags(listCmd, flag.FlagSet{
-		Optional: []flag.Flag{flag.NamespaceCanary},
+		Optional: []flag.Flag{flag.NamespaceCanary, flag.Namespace, flag.AllNamespaces},
```

Comment:
If we are setting `flag.Namespace` across all commands here, we can use the global option for the `command ` variable below.


```
wrapper.SetFlags(command, flag.FlagSet{
		Global: []flag.Flag{flag.Namepace},
	})
```

So that we don't need to make changes across all commands.